package/Dockerfile

```diff
@@ -10,6 +11,10 @@ FROM base AS copy_buildx
 ONBUILD ARG TARGETARCH
 ONBUILD COPY bin/fleetcontroller-linux-${TARGETARCH} /usr/bin/fleetcontroller
 
+FROM base AS copy_goreleaser
+ONBUILD ARG ARCH
+ONBUILD COPY fleetcontroller-linux-${ARCH} /usr/bin/fleetcontroller
```

Comment:
I guess this is needed because of `no_unique_dist_dirs`?

Previously: https://github.com/rancher/fleet/blob/fb18a5e507c5c75de761bc8d73b38be45e8104da/package/Dockerfile#L11-L12

Reply:
Correct.